conjur/client.py

```diff
@@ -81,10 +80,9 @@ def __init__(self,
                         on_disk_config[field_name] = field_value
                 loaded_config = on_disk_config
 
+            # TODO add error handling for when conjurrc field doesn't exist
             except Exception as exc:
-                raise ConfigException(f"Error: {exc} not found in " \
-                                      "the loaded conjurrc\n") from Exception
-
+                raise ConfigException(exc) from Exception
```

Comment:
Isn't it a bug?
Shouldn't it be like this?
```suggestion
                raise ConfigException(exc) from exc
```

Reply:
For this you need code coverage tests that ensure you write interpretable code that behave as you intended.

Reply:
I do think so because otherwise the ConfigException is not being cascaded up. Changing